### ISSUE TITLE
Cannot create cloud_watch_event_target

### ISSUE THREAD

**Issue by seed-doordash** [localstack/localstack#7661]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

environment
-------------------------------
**aws-cli**
aws-cli/1.27.69 Python/3.10.8 Darwin/22.3.0 botocore/1.27.96

**localstack**
localstack --version (installed via pip)
1.3.1

Behaviour
--------------------------
stuck at 
module.service.module.resource_record_streamer_local[0].aws_cloudwatch_log_group.lambda[0]: Still creating... [3m10s elapsed]

The Error in debug log
-------------------------------
```
Traceback (most recent call last):
File "/opt/code/localstack/localstack/aws/protocol/parser.py", line 172, in wrapper
return func(*args, **kwargs)
File "/opt/code/localstack/localstack/aws/protocol/parser.py", line 895, in parse
operation = self.service.operation_model(operation_name)
File "/opt/code/localstack/.venv/lib/python3.10/site-packages/botocore/utils.py", line 1419, in _cache_guard
result = func(self, *args, **kwargs)
File "/opt/code/localstack/.venv/lib/python3.10/site-packages/botocore/model.py", line 364, in operation_model
raise OperationNotFoundError(operation_name)
botocore.model.OperationNotFoundError: ListTagsForResource

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
File "/opt/code/localstack/localstack/aws/chain.py", line 90, in handle
handler(self, self.context, response)
File "/opt/code/localstack/localstack/aws/handlers/service.py", line 63, in __call__
return self.parse_and_enrich(context)
File "/opt/code/localstack/localstack/aws/handlers/service.py", line 76, in parse_and_enrich
operation, instance = parser.parse(context.request)
File "/opt/code/localstack/localstack/aws/protocol/parser.py", line 176, in wrapper
raise UnknownParserError(
localstack.aws.protocol.parser.UnknownParserError: An unknown error occurred when trying to parse the request.
```

### Steps To Reproduce

	python3 -m pip install localstack
	SERVICES=dynamodb,sqs,eventbridge,iam,s3,lambda,events,cloudwatchlogs,cloudwatch DEBUG=1 localstack start -d

### Environment

```markdown
- OS:
❯ sw_vers
ProductName:		macOS
ProductVersion:		13.2
BuildVersion:		22D49

- LocalStack: 1.3.1
```
python3 -m pip install localstack

### Anything else?

I had to first install awscli via `pip install awscli` which installed botocore==1.29.69 
Then when i installed localstack 1.3.1, via `pip install localstack`, it uninstalled the botocore and override it with version 1.27.96.

This seems to cause this error.
How do i get around?

**Comment by seed-doordash** [comment]
Essentially localstack is overriding the botocore version even when i do pip install -I botocore.

**Comment by BazzalSeed** [comment]
The walkaround/solution for me now is to
1. uninstall awscli (from pip or brew)
2. install awscli-local via pip 
3. install and run localstack

**Comment by steffyP** [comment]
Hi @seed-doordash,

thanks for reporting. In the meantime we have released localstack version `1.4.0` and the issue should be resolved.
Could you please confirm that it is working for you?
